README.md

```diff
@@ -329,6 +329,16 @@ if:
     - "status-name-2"
     - "status-name-3"
 
+  # "has_workflow_result" is satisfied if the GitHub Actions workflow runs that
+  # are specified all finished and concluded with one of the conclusions
+  # specified. "conclusions" is optional and defaults to ["success"].
+  # `workflows` contains the paths to the workflow files that are being checked.
+  has_workflow_result:
+    conclusions: ["success", "skipped"]
+    workflows:
+      - ".github/workflows/a.yml"
+      - ".github/workflows/b.yml"
```

Comment:
Since workflows must exist in the `.github/workflows` directory, would it make sense to exclude that from these paths? Or do you think it is clearer to specify the full path?

Reply:
I had a look at both, and for me having the full path makes it clearer. I find

```
- a.yml
- b.yml
```

to be a tiny bit more obscure. It's harder to get confused about which files we're talking about if it's a full path.

But it doesn't matter _that much_. WDYT?

Reply:
Using full paths sounds fine to me then